src/main/java/com/google/codeu/data/Message.java

```diff
@@ -70,32 +72,22 @@ public Message(UUID id, String user, long timestamp, String text) {
   /**
    * Converts list of comment IDs as strings to list of comment IDs as UUIDs.
    */
-  public List<UUID> convertStringsToCommentIds(List<String> commentIdsAsStrings) {
-    if (commentIdsAsStrings == null) {
-      return new ArrayList<>();
-    }
+  public List<UUID> convertStringsToUUIDs(List<String> uuidStrings) {
```

Comment:
```suggestion
  public List<UUID> convertStringsToUuids(List<String> uuidStrings) {
```
Should fix the error

Reply:
yup fixed already!